server/services/receive.js

```diff
@@ -0,0 +1,161 @@
+const bunyan = require('bunyan');
+const path = require('path');
+
+const log = bunyan.createLogger({ name: 'receive-service' });
+
+const Response = require(path.join(__dirname, './response'));
+const GraphAPi = require(path.join(__dirname, './graph-api'));
+const Rescue = require(path.join(__dirname, './rescue'));
+const Update = require(path.join(__dirname, './update'));
+
+const has = require('has');
```

Comment:
Minor but better to group related imports (In this case, bunyan, path, and has are all libraries)